### DISCUSSION TITLE
Impossible to create object (410: 410 Gone)

### DISCUSSION THREAD

**Discussion by lmilbaum** [python-gitlab/python-gitlab#2571] [Q&A] [answered]
Running the following command raises an error which is unclear. The intention is to lint the local gitlab ci file.
```
[lmilbaum@lmilbaum rh-renovate-image]$  gitlab --verbose ci-lint create --content @.gitlab-ci.yml
Impossible to create object (410: 410 Gone)
```

**Comment by nejch** [comment, marked as answer]
It seems like this was removed in GitLab itself, hence the `410 Gone`:
https://docs.gitlab.com/ee/api/lint.html#validate-the-ci-yaml-configuration-deprecated

For now we should probably switch to the `project-ci-lint create` command and stop documenting the old one:
https://python-gitlab.readthedocs.io/en/stable/cli-examples.html

And then also remove the endpoint entirely in the code for GitLab 16.0+.

**Reply by lmilbaum** [reply]
Thanks
